### ISSUE TITLE
Captives at Deadholmes quest, Apothecary Enith is missing!

### ISSUE THREAD

**Issue by Cemow** [dalaranwow/dalaran-wow#5542]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
In the quest named "Captives at Deadholmes", the NPC named "Apothecary Enith" is not in the place where he should be. Location Ghostlands
**Expected behaviour**: Tell us what should happen instead
He should be laying down at the botttom of the crypt.
**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Cemow** [comment]
The quest named "Captives at Deadholmes" is bugged. Apothecary Enith is not in the place where he should be.

**Comment by boujii** [comment]
Indeed he should be laying on the bottom of the floor in the crypt. 
Here are a couple pictures from my developer server to ensure you are in the right place.

https://i.imgur.com/0UU5QYU.png
https://i.imgur.com/eY1j4Ry.png

Spawntime is every 600 seconds.

**Comment by Golganeth** [comment]
The NPC is there.

![image](https://user-images.githubusercontent.com/5057942/44310714-1b8a1680-a3db-11e8-8c10-97375df30ec9.png) 

Closing.
